staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -379,6 +397,10 @@ type JobStatus struct {
 	// +optional
 	Failed int32 `json:"failed,omitempty" protobuf:"varint,6,opt,name=failed"`
 
+	// The number of pods which are terminating.
```

Comment:
```suggestion
	// The number of pods which are terminating (in phase Pending or Running and have a non-nil deletionTimestamp).
```

Also explain that this is alpha and needs the feature gate to be enabled.